extensions/sql-database-projects/src/models/project.ts

```diff
@@ -195,6 +187,7 @@ export class Project {
 		}
 		else {
 			this.projFileXmlDoc.documentElement.appendChild(importNode, oldImportNode);
+			this.importedTargets.push(project);	// Add new import target to the list
```

Comment:
I may be missing it, but what does `this.importedTargets` end up getting used to do?  At my first glance, it appears to be unused.

Reply:
For now, it is just used for the purpose of verifying what all "Import" statements are there in the sqlproj. And based on that, if we need to update the sqlproj (which will be true for projects originating in SSDT). This will also be helpful if we plan on adding a feature to "Reload" the project.